packages/client/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,11 @@
+import './index.css';
+
+export default (props: { image: string; className: string }) => {
```

Comment:
Тут можно сразу props деструктурировать 
```suggestion
export default ({ name, classname }: { image: string; className: string }) => {
```